lib/livebook_web/session_identity.ex

```diff
@@ -1,9 +1,9 @@
-defmodule LivebookWeb.Cookies do
+defmodule LivebookWeb.SessionIdentity do
   # This module implements the ZTA contract specific to Livebook cookies
   @moduledoc false
 
-  def authenticate(_, _conn) do
-    %{}
+  def authenticate(_, _, _) do
+    %{id: Livebook.Utils.random_id()}
   end
```

Comment:
We actually want to store and retrieve the id from session, otherwise it's random every time we call `authenticate`?